beetsplug/missing.py

```diff
@@ -204,10 +252,10 @@ def _missing_albums(self, lib, query):
             if total:
                 continue
```

Comment:
I'm pretty sure this filter is broken now, since it's comparing release group and release IDs. In my opinion, the right thing would to indeed filter by release group. But again, someone who actually uses the plugin should comment.

Reply:
I've restored this lines, now it's comparing release group with release group as it should be